tgui/packages/tgui/interfaces/Orbit.js

```diff
@@ -69,6 +76,25 @@ const OrbitedButton = (props, context) => {
   return (
     <Button
       color={color}
+      tooltip={
+        thing.assigned_role ? (
+          <Flex>
+            <Box
+              as="img"
+              mr="0.5em"
+              // mt={0.5}
+              className={classes([
+                'orbit_job16x16',
+                thing.assigned_role_sprite,
+              ])}
+            />{' '}
+            {thing.assigned_role}
+          </Flex>
```

Comment:
```suggestion
          <Stack>
            <Box
              as="img"
              mr="0.5em"
              className={classes([
                'orbit_job16x16',
                thing.assigned_role_sprite,
              ])}
            />{' '}
            {thing.assigned_role}
          </Stack>
```
I don't see the point to comment that, and use Flex instead already imported Stack